backend/module/chaos/experimentation/xds/internal/xdstest/xdstest.go

```diff
@@ -33,9 +35,13 @@ func NewTestModuleServer(c func(cfg *any.Any, logger *zap.Logger, scope tally.Sc
 		}
 	}
 
+	if config.CacheRefreshInterval == nil {
+		config.CacheRefreshInterval = durationpb.New(time.Second * 1)
+	}
```

Comment:
It's not necessary per se - I can move it to the caller-side. I just found it helpful to not have to specify cache refresh interval on the caller side since it's almost always required in practice (for newly added tests)